drivers/sdcard/sdcard.py

```diff
@@ -231,15 +231,15 @@ def readblocks(self, block_num, buf):
         assert nblocks and not len(buf) % 512, "Buffer length is invalid"
         if nblocks == 1:
             # CMD17: set read address for single block
-            if self.cmd(17, block_num * self.cdv, 0, release=False) != 0:
+            if self.cmd(17, block_num * self.cdv, 0x3B, release=False) != 0:
```

Comment:
Won't the CRC here depend on the value of `block_num * self.cdv`?  Same with the 3 other lines below.